.circleci/config.yml

```diff
@@ -56,15 +56,38 @@ jobs:
 
       - run:
           working_directory: content
-          command: ./scripts/deploy.sh
+          command: ./scripts/upload.sh
+
+  website-warm-cache-check-links:
+    docker:
+      - image: *MIDDLEMAN_IMAGE
+    steps:
+      # Don't need gems or anything
+      - run:
+          name: Warm cache and check for broken links
+          command: wget --delete-after --level inf --no-directories --no-host-directories --no-verbose --page-requisites --recursive --spider "https://www.terraform.io/"
```

Comment:
Do you want to call `./scripts/warm-cache.sh` here instead?

Reply:
I changed that in the latest commit. The initial version of this job ran checkout, then ran the script from the repository... but then I realized we were checking out the entire repository for literally just one `wget` command, and decided we could probably just inline it. That might have been premature optimization, since I'm not sure what Circle does to make checkouts performant! But, since I'm hoping to replace wget with something else here soonish... seems fine either way.